content/docs/operations/debug.md

```diff
@@ -57,37 +52,40 @@ Make sure to use the version of `{{< reuse "docs/snippets/cli-name.md" >}}` that
-   ```
+   ```bash
+   # kgateway control plane
+   kubectl logs -n {{< reuse "docs/snippets/ns-system.md" >}} deployment/kgateway
    
-   Example output for open source: 
-   ```console
-   NAME                                     READY   STATUS    RESTARTS   AGE
-   gloo-6bf6478bd8-rdghl                    1/1     Running   0          141m
-   gloo-proxy-http-77454f4df9-vchvf         2/2     Running   0          140m
-   redis-54757c7964-zxnb8                   1/1     Running   0          141m
+   # Replace $GATEWAY_NAME with the name of your gateway.
+   kubectl logs -n {{< reuse "docs/snippets/ns-system.md" >}} deployment/gloo-proxy-$GATEWAY_NAME
    ```
    
-   Example output for Enterprise edition:
-   ```console
-   NAME                                     READY   STATUS    RESTARTS   AGE
-   extauth-7f67b6bc68-2xsth                 1/1     Running   0          141m
-   gloo-6bf6478bd8-rdghl                    1/1     Running   0          141m
-   gloo-proxy-http-77454f4df9-vchvf         2/2     Running   0          140m
-   rate-limit-5fc5fc9b8c-g4mfm              1/1     Running   0          141m
-   redis-54757c7964-zxnb8                   1/1     Running   0          141m
+<!-- TODO: CLI
+## Before you begin
+
+If you have not done yet, install the `{{< reuse "docs/snippets/cli-name.md" >}}` CLI. The `{{< reuse "docs/snippets/cli-name.md" >}}` CLI is a convenient tool that helps you gather important information about your gateway proxy. To install the `{{< reuse "docs/snippets/cli-name.md" >}}`, you run the following command: 
+```sh
+curl -sL https://run.solo.io/gloo/install | sh
+export PATH=$HOME/.gloo/bin:$PATH
+```
+
+{{% callout type="info" %}}
+Make sure to use the version of `{{< reuse "docs/snippets/cli-name.md" >}}` that matches your installed version.
+{{% /callout %}}
+
+-->
+
+<!-- TODO: CLI
+5. Check the proxy configuration that is served by the {{< reuse "docs/snippets/product-name.md" >}} xDS server. When you create {{< reuse "docs/snippets/product-name.md" >}} resources, these resources are translated into Envoy configuration and sent to the xDS server. If {{< reuse "docs/snippets/product-name.md" >}} resources are configured correctly, the configuration must be included in the proxy configuration that is served by the xDS server. 
+   ```sh
+   {{< reuse "docs/snippets/cli-name.md" >}} proxy served-config --name gloo-proxy-http
    ```
-   
-7. Review the logs for each component. Each component logs the sync loops that it runs, such as syncing with various environment signals like the Kubernetes API. You can fetch the latest logs for all the components with the following command. 
+
+6. Review the logs for each component. Each component logs the sync loops that it runs, such as syncing with various environment signals like the Kubernetes API. You can fetch the latest logs for all the components with the following command. 
    ```bash
-   {{< reuse "docs/snippets/cli-name.md" >}} debug logs
+   glooctl debug logs
```

Comment:
should this be removed?

Reply:
I left some of the old CLI content in comments with a TODO, but I can also just remove it if you think that would be better?